### ISSUE TITLE
An adaptation problem related to layout

### ISSUE THREAD

**Issue by yuhuitech** [commons-app/apps-android-commons#4548]
**Describe the bug**

Dear developer!
In the process of using, I found a bug in layout / display. As shown in the figure below, on the log-in page, **the logo and a part of texts tangled together**. This may be an adaptation problem. I assume it is because **display and font size** are set to large **(mainly due to display size)**.
Because of some problems with my eyesight, I often use the largest font and display.
![103165857-dbaf8580-4857-11eb-9e22-120ae1901389](https://user-images.githubusercontent.com/25502419/128597553-7949a1a3-0b21-4ddf-9357-766b5c302518.png)

App version is 3.0.3, my mobile phone is Google Pixel 2, and the screen resolution setting is 1920* 1080. And **_my font size and display size are the largest ! ! !_** Also, my Android system version is 10.0.
The above is a description of the problem. The overall experience of the software is good. You can see that the design is very attentive. Thank you very much for the efforts of the developers all the time. I hope to receive a reply. Thank you so much!!

**Expected behavior**

I hope that the logo on the log-in page is separated from the text below.

**Steps to reproduce the behavior:**

1.use the largest font and display in the display setting
2.open this app
3.this error could be seen

**Screenshot**

<img src="https://user-images.githubusercontent.com/25502419/129313218-e1aade95-ba04-4512-92c7-a51e81ab8f76.png" alt="copy" width="250"/>

**Environment**

1. Version of Common: 3.0.3
2. Device : Google Pixel 2,
3. OS version : Android 10.0

**Comment by nicolas-raoul** [comment]
Thank you for the report, please do not hesitate to report similar problems! :-)

Strangely, I can already scroll using the latest master:
![adb](https://user-images.githubusercontent.com/99590/129356631-960d1491-9090-4d7b-bf53-f5b7d5657b38.png)

Did you observe the problem (no scrolling) on the latest master?
Thanks!

**Comment by yuhuitech** [comment]
Thanks! :) I ran the code form the latest master branch. In the landscape mode, this page is scrollable and this problem doesn't exist. However, in the portrait mode, this problem occurs and this page is not scrollable.

**Comment by nicolas-raoul** [comment]
I was able to reproduce the problem on a super-small screen. Indeed there is no scrolling in portrait mode.
![Screenshot_1628915532](https://user-images.githubusercontent.com/99590/129434004-02239516-8f08-4a92-9c40-1a7faaa4d2fe.png)

**Comment by yuhuitech** [comment]
Yes, this problem often occurs on a small screen. And in portrait mode, this page is not scrollable.
